examples/isomorphic/dev.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd $(dirname $0)
+
+cd ./client
+
+./build-wasm.dev.sh &
```

Comment:
Will all of these processes be killed if you hit Ctrl-C here? Doesn’t seem like it but maybe I’m missing something?

Reply:
Maybe something like:

```
#!/bin/bash
 cd $(dirname $0)
 cd ./client

./build-wasm.dev.sh &

systemfd --no-pid -s http::7878 -- cargo +nightly watch -w ../ -x 'run -p isomorphic-server' &

trap "exit" INT TERM
trap "kill 0" EXIT

wait
```

But my bash is weak so YMMV ..

Reply:
My bash is **so weak** you have no idea!
But they did die for me.
The server's graceful shutdown causes a collision for the `systemfd`'s port if I press Ctrl-C and then immediately start the script again, but one would not be doing this as everything reloads automatically, so why restart it by hand. Give it a couple of seconds and it works always.
Also no problems with pressing Ctrl-C inside a build.

Reply:
`systemfd` I just took from here: https://actix.rs/docs/autoreload/
I knew of `catflap` before though.

Reply:
Hmm honestly I might still add the traps for safety. Not sure if background scripts die when the parent script dies on all OS's but again - I'm not sure ha..!

Reply:
So, those go *before* wait? Where did you get them from?

Reply:
Found it here https://stackoverflow.com/a/22644006

Then tested it by running 

```
#!/bin/bash

(sleep 5 && echo hi) &
(sleep 2 && echo bar) &

trap "exit" INT TERM
trap "kill 0" EXIT

wait
```

If you run it and `Ctrl-C` you won't get any output

If you remove the traps and run it and `Ctrl-C` those echos will still run.

Reply:
The [4th](https://stackoverflow.com/questions/360201/how-do-i-kill-background-processes-jobs-when-my-shell-script-exits/22644006#comment72425270_22644006) and [5th](https://stackoverflow.com/questions/360201/how-do-i-kill-background-processes-jobs-when-my-shell-script-exits/22644006#comment75666269_22644006) comments are really concerning.
Maybe I should use npm-run-all :joy: :ok_hand: since we definitely have node (for webpack)